### PR TITLE
Changed arguments

### PR DESCRIPTION
**Pull request by zce**
cjpatoilo/rasper#4

### CODE REVIEW

**Comment by wolves** [comment]
cc: @zce @cjpatoilo 

Unfortunately, this only prepends the first file when the cli completes.

**Example**
If there are the following files in `dist/`: 

```
a.css
a.min.css
b.css
b.min.css
```
The issue is that the `dist/*.css` argument in `$ banner-cli dist/*.css` is evaluated and not simply passed through as a string resulting in:
```
process.argv
# => {  _: [ 'a.css', 'a.min.css', 'b.css', 'b.min.css'], ... }
```
Rather than:
```
process.argv
# => {  _: [ 'dist/*.css'], ... }
```
This leads to this conditional
https://github.com/cjpatoilo/banner-cli/blob/86e4a2c2e7462c6679e6e6710e11a06c636f1202/cli.js#L87-L90

setting `options.source = 'a.css'` rather than `options.source = 'dist/*.css'`

This is fine until later here:
https://github.com/cjpatoilo/banner-cli/blob/86e4a2c2e7462c6679e6e6710e11a06c636f1202/index.js#L25-L29

The call to `glob` receives `a.css` for its pattern argument rather than the intended `dist/*.css`.
This leads to all the remaining CSS files within the dist directory being ignored.

**In Summary:**

Your change fixes part of the issue where previously the template was simply being applied to the `node_modules/.bin/banner-cli` file itself, but we need to figure out a better way to handle glob args passed in via the command line.

One way to make it work with just this change is wrapping the glob arg in quotes like this:
`banner-cli "dist/*.css"`
but that doesn't feel intuitive and I think a better solution can be applied.

I am working on it currently.
